swarm/api/manifest.go

```diff
@@ -220,6 +222,10 @@ func readManifest(manifestReader storage.LazySectionReader, hash storage.Key, dp
 		err = fmt.Errorf("Manifest not Found")
 		return
 	}
+	if size > manifestSizeLimit {
```

Comment:
Maybe add an error log with the key `hash`?

Reply:
👍 Added